src/scss/_header.scss

```diff
@@ -71,41 +71,94 @@
     display: none;
 
     @media screen and (max-width: $xs) {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
       position: relative;
-      display: block;
+
+      transition: 0.5s;
+
       width: 30px;
       height: 20px;
-      border-top: 4px solid $color__gray-99;
-      border-bottom: 4px solid $color__gray-99;
 
       &:before {
-        position: absolute;
         content: '';
-        top: 4px;
-        left: 0px;
+        height: 4px;
         width: 100%;
-        border-top: 4px solid $color__gray-99;
+
+        transition: 0.5s;
+
+        transform-origin: left;
+
+        background-color: $color__gray-99;
       }
 
-      &:hover {
-        display: none;
+      &:after {
+        content: '';
+        height: 4px;
+        width: 100%;
+
+        transition: 0.5s;
+
+        transform-origin: left;
+
+        background-color: $color__gray-99;
+      }
+
+      &:focus {
+        &::before {
+          transform: rotate(33deg);
+        }
+
+        &::after {
+          transform: rotate(-33deg);
+        }
+
+        span {
```

Comment:
don't use tag as selector

Reply:
For some reason it doesn't want to work like this.
![q1](https://user-images.githubusercontent.com/62384781/192161540-8002f602-d5d1-44a4-834b-1db6168b4605.png)
![q2](https://user-images.githubusercontent.com/62384781/192161543-e1433895-9cd0-46a4-8102-490018520b4e.png)